package.json

```diff
@@ -8,8 +8,13 @@
     "test": "jest --forceExit --detectOpenHandles",
     "testFront": "jest './test/reactTests.test.js' --coverage --forceExit --detectOpenHandles",
     "testBack": "jest './test/mongodbTests.test.js' --forceExit --detectOpenHandles",
+    "testSeedGeneration": "jest './test/dataGeneration.test.js' --forceExit --detectOpenHandles",
     "seed": "node database-mongodb/seed.js",
-    "start": "nodemon start.js "
+    "start": "nodemon start.js ",
+    "postgres-generate": "node database-postgres/generateDescriptions.js & node database-postgres/generateDetails.js & node database-postgres/generateAttributes.js & node database-postgres/generateDirections.js",
+    "postgres-seed-descriptions": "knex seed:run --specific=1seedDescriptions.js",
+    "parrallel-postgres-seed": "knex seed:run --specific=3seedDetails.js & knex seed:run --specific=2seedAttributes.js & knex seed:run --specific=4seedDirections.js",
```

Comment:
Whats are the differences between `parrallel-postgres-seed' and `postgres-seed`?

Reply:
I changed my functionality around. But my reasoning was that parrallel-postgres-seed would seed all the tables that depended on my descriptions seed **after.** descriptions table was seeded.  postgres-seed says "seed the descriptions **first**, and **then** seed all the tables that depend on the description, which has the foreign key data needed for the other tables, **after** descriptions is done seeded. Since the other tables don't depend on one another, only the descriptions table, they don't need to be loaded sequentially.